packages/docs/docs/how-to-use.mdx

```diff
@@ -2,135 +2,226 @@
 title: How to Use
 ---
 
-# How to use
+# How to Use
+
+## Accounts
+
+### View all accounts
+
+You can view all of your accounts by clicking on the down caret underneath the account icon, or clicking on "Accounts" in the navigation sidebar.
+
+<div style={{ display: 'flex', gap: '20px', flexWrap: 'wrap' }}>
+  <img alt="navigation" src="/wallet/wallet-accounts-button.png" height="500" />
+  <img alt="navigation" src="/wallet/wallet-screen-nav.png" height="500" />
+  <img alt="navigation accounts" src="/wallet/nav-accounts.png" height="500" />
+</div>
+
+### Add a new account
+
+You can add a new account on the accounts page by clicking on the "Add new account" or "Add from private key" buttons.
+
+<div style={{ display: 'flex', gap: '20px', flexWrap: 'wrap' }}>
+  <img alt="accounts screen" src="/wallet/accounts-screen.png" height="500" />
+  <img
+    alt="add account screen"
+    src="/wallet/add-account-screen.png"
+    height="500"
+  />
+  <img
+    alt="import account screen"
+    src="/wallet/import-account-screen.png"
+    height="500"
+  />
+</div>
+
+### Edit an account
+
+To edit an account, click on the horizontal ellipsis to the right of the account and select "Edit".
+
+<div style={{ display: 'flex', gap: '20px', flexWrap: 'wrap' }}>
```

Comment:
We should create a component for this to avoid code repetition. Also we can add an automatic scroll on the y-axis to prevent images from being broken in lines.